src/sonic-host-services/scripts/caclmgrd

```diff
@@ -399,7 +399,8 @@ class ControlPlaneAclManager(daemon_base.DaemonBase):
 
                 acl_rules = {}
 
-                for ((rule_table_name, rule_id), rule_props) in self._rules_db_info.items():
+                for ((rule_table_name, rule_id), rule_props) in self._rules_db_info.iteritems():
```

Comment:
Why change `.items()` to `.iteritems()`? This file is now run with Python 3; `.iteritems()` no longer exists. This may be the root of the test failure?

Reply:
The change was not intentional. When trying to resolve merge conflict, the code got changed mistakenly. I will correct it and submit.